src/webviews/apps/tsconfig.json

```diff
@@ -17,6 +17,8 @@
 		"../**/protocol.ts",
 		"../../config.ts",
 		"../../constants.ts",
+		"../../emojis.ts",
+		"../../emojis.generated.ts",
```

Comment:
Note @eamodio @ericf-axosoft I updated the webview tsconfig because these modules are now being used inside the webviews folder (in GraphWrapper specifically). This eliminates the following build warnings:

```
  WARNING in ../../emojis.generated.ts
  Module Warning (from ../../../node_modules/esbuild-loader/dist/index.cjs):
  [esbuild-loader] The specified tsconfig at "C:\Projects\GitLens\vscode-gitlens\src\webviews\apps\tsconfig.json" was applied to the file "C:\Projects\GitLens\vscode-gitlens\src\emojis.generated.ts" but does not match its "include" patterns

  WARNING in ../../emojis.ts
  Module Warning (from ../../../node_modules/esbuild-loader/dist/index.cjs):
  [esbuild-loader] The specified tsconfig at "C:\Projects\GitLens\vscode-gitlens\src\webviews\apps\tsconfig.json" was applied to the file "C:\Projects\GitLens\vscode-gitlens\src\emojis.ts" but does not match its "include" patterns
```

![image](https://github.com/gitkraken/vscode-gitlens/assets/67011668/e7c0bdff-fe4b-4c54-b28a-5d0f53a5167a)